packages/patterns-react/src/patterns/SimpleLongForm/__stories__/SimpleLongForm.stories.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text, select, object } from '@storybook/addon-knobs';
+import '../../../../../styles/scss/patterns/simplelongform/index.scss';
+import SimpleLongForm from '../SimpleLongForm';
+import readme from '../README.md';
+
+storiesOf('Simple Long Form', module)
```

Comment:
Since this is in alpha, wrap the stories with the feature flag:

```javascript
if(SIMPLELONGFORM) {
    storiesOf('Simple Long Form', module)
    // stories here...
}
```

Reply:
updated. Thanks